tests/test_optimizer.py

```diff
@@ -399,6 +399,8 @@ def test_qualify_columns(self, logger):
             'WITH "t" AS (SELECT 1 AS "c") (SELECT "t"."c" AS "c" FROM "t" AS "t")',
         )
 
+        # Struct expansion tests
```

Comment:
Consider moving these tests to `qualify_columns.sql`. Check out line 477 and below in said file.